docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -2,18 +2,23 @@
 
 The parent key for all of the following parameters is `openshift4_olm`.
 
-== `namespace`
```

Comment:
I think namespace is still configurable.

Reply:
Removed the configurable part.